guidelines/sc/20/visual-presentation.html

```diff
@@ -9,9 +9,9 @@ <h4>Visual Presentation</h4>
    <ul>
      <li>Foreground and background colors can be selected by the user.</li>   
      <li>Width is no more than 80 characters or glyphs (40 if CJK).</li>   
-     <li>Text is not justified (aligned to both the left and the right margins).</li>   
+     <li><a>Text</a> is not justified (aligned to both the left and the right margins).</li>   
```

Comment:
Since there is an included change in this PR to include a link to the definition of text in "Block of text" (which is a defined term in the preamble of this SC) , it seems redundant and unnecessary to also link to the definition of text in a bullet of the same SC.
```suggestion
     <li>Text is not justified (aligned to both the left and the right margins).</li>   
```

Reply:
Images of text could use full justification (and the other features listed in 1.4.8 Visual Presentation might be achieved using images of text) -- so I agree this use should not link to the defined term.